docs/topics/developer-lightspeed/proc_configuring-llm-podman-desktop.adoc

```diff
@@ -0,0 +1,58 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-llm-podman_{context}"]
+= Configuring the LLM in Podman Desktop
+
+[role="_abstract"]
+
+The Podman AI lab extension enables you to use an open-source model from a curated list of models and use it locally in your system. 
```

Comment:
Covering instructions for Podman AI sounds useful.
There is a concern that for practical purposes it is HIGHLY likely that users will be disappointed/frustrated running local models.  This is a nature that the majority of models run locally are smaller and not powerful, that directly limits the potential help they can provide to MTA.

For most purposes a local model will yield poor results.  

We likely want to be clear to instruct the user that MTA's behavior is related to the capability of a model, often local models are smaller and less powerful, therefore they often lack the ability to significantly help with MTA use-cases.